homework/src/app.js

```diff
@@ -1,11 +1,184 @@
-{
-  /* Your homework goes here */
   
   /* STRINGS */
 
+  console.log(`------ STRINGS -------`);
+  console.log(`\n`);
+
+  const myString = "hello,this,is,a,difficult,to,read,sentence";
+  console.log(myString);
+  console.log(`The length of myString is: ${myString.length}`);
+
+  const myNewString = myString.replace(/,/g , ` `); // g(global) is used to replace all occurances of ',' with ' '
+  console.log(myNewString);
+  console.log(`\n`);
+
+  const str = 'dlroW olleH';
+  console.log(`Old string: ${str}`);
+
+  const arr = str.split(``);
+  arr.reverse();
+  const newStr = arr.toString();
+  const reversedStr = newStr.replace(/,/g , ``);
```

Comment:
^ you can remove `reversedStr`, it is not used in this exercise.

Reply:
I made the correction accordingly. Thank you.